framework/doc/content/source/transfers/MultiAppInterpolationTransfer.md

```diff
@@ -1,8 +1,25 @@
 # MultiAppInterpolationTransfer
 
-The MultiAppInterpolationTransfer transfers the source variable to the nearest node on the
+The `MultiAppInterpolationTransfer` transfers the source variable to the nearest node on the
 target mesh using mesh interpolation, including the ability to utilize the displaced
-configuration for either or both the source and target.
+configuration for either or both the source and target.  
+
+Nodal transfers using the default settings for this interpolation scheme, `interp_type=inverse_distance` and `num_points=3`, are 
```

Comment:
I am not sure I am convinced by "num_points=3". I could imagine it is problem dependent  ...

Reply:
He's just saying `num_points=3` because that is the default

Reply:
Oh, I see. What I meant was that it was hard to say the default setting work best (`best suited` ) for something.

Reply:
I changed the wording around this.